src/models/pang14/README.md

```diff
@@ -0,0 +1,57 @@
+# Pangolin (pang14)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Pangolin](./img/pang14.webp)
+
+The System76 Pangolin is a laptop with the following specifications:
+
+- CPU
+    - Supports AMD Ryzen (Rembrandt) Mobile CPUs
```

Comment:
I believe this is a different CPU generation from the pang12 and pang13?

Reply:
Looks like it should be Phoenix.

Reply:
Yes, good catch.